internal/stage_6.go

```diff
@@ -72,9 +72,9 @@ func createTcpConn(destination string) (net.Conn, error) {
 
 func sendRequestDirectlyOverTcp(logger *logger.Logger, conn net.Conn, i int) error {
 	req := "GET / HTTP/1.1\r\n" + "\r\n\r\n"
-	logger.Infof("Sending request on %d (status line): %s", i, getFirstLine(string(req)))
+	logger.Infof("Sending request on %d (status line): %q", i, getFirstLine(string(req)))
```

Comment:
@libmartinito can you show me an example of what this looks like in logs? Does it also print `\r\n` for example? If we're saying this is the first line, then I don't think it should include a `\r\n`.

Reply:
This one is not going to be super helpful to use `%q` here.